SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -1101,3 +1109,33 @@ void USpatialSender::UpdateInterestComponent(AActor* Actor)
 	Worker_EntityId EntityId = PackageMap->GetEntityIdFromObject(Actor);
 	Connection->SendComponentUpdate(EntityId, &Update);
 }
+
+void USpatialSender::RequestEntityDeletion(const Worker_EntityId EntityId, const bool bForceDelete /* = false */)
+{
+	const AActor* Actor = Cast<AActor>(PackageMap->GetObjectFromEntityId(EntityId).Get());
+
+	if (Actor &&
+		Actor->IsNetStartupActor() &&
+		Actor->GetIsReplicated() &&
+		bForceDelete == false)
+	{
+		// In the case that this is a startup actor, we won't actually delete the entity in SpatialOS.  Instead we'll Tombstone it.
+		MarkEntityTombstone(EntityId);
```

Comment:
Potential optimisation, when we activate the tombstone, should we remove older, redundant components?

Reply:
yes, i was thinking that too as a second pass done with Dynamic Components.